data/json/recipes/recipe_others.json

```diff
@@ -7904,5 +7904,36 @@
     [ [ "wheel", 1 ] ],
     [ [ "motor", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "neoprene",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_OTHER",
+  "skill_used": "cooking",
+  "skills_required": [ "cooking", 4 ],
+  "difficulty": 4,
+  "time": 25000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [["adv_chemistry", 3] , ["textbook_chemistry", 3]],
+  "qualities":[
+    {"id":"CHEM","level":2,"amount":1}
+  ], "tools": [
+    [
+      [ "hotplate", 100 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 100 ],
+      [ "fire", -1 ],
+      [ "chemistry_set", 100]
+    ]
+  ],
+  "components": [
+    [ [ "chem_ethanol", 2 ] ],
+    [ [ "scrap_copper", 1 ] ],
```

Comment:
I think that you could use 100 units of "copper" item instead of 1 unit of scrap copper, as you can disassemble 1 scrap copper into 100 copper.